frontend/src/metabase/visualizations/components/FunnelNormal.styled.tsx

```diff
@@ -61,11 +61,11 @@ export const FunnelStart = styled.div<SharedProps>`
   flex-grow: 1;
 
   padding-right: 0.5em;
-  font-size: 24px;
+  font-size: 1.72em;
 
   ${Title} {
     font-weight: bold;
-    color: black;
+    color: var(--mb-color-text-dark);
```

Comment:
This is intentional. Checked with @kdoh that we should change this to `text-dark` as opposed to creating a single `text-black` that is only used once in a single viz.